scripts/hostapd/dnsmasq.conf

```diff
@@ -1,2 +1,2 @@
-dhcp-range=10.0.0.2,10.0.0.127,255.255.255.0,12h
+dhcp-range=10.0.0.2,10.0.0.254,4h
```

Comment:
I am DHCP-ing the full range (no room for static) and changing to 4h lease.

Reply:
```dhcp-range=10.0.0.2,10.0.0.254,4h```

should be 

```dhcp-range=10.0.0.2,10.0.0.254,255.255.255.0,4h```

So there is no confusion on the assigned ip addresses that they are /24  on the client side